<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home_impiegato_window</class>
 <widget class="QMainWindow" name="home_impiegato_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>259</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>259</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>(Gestione) Portale Tipografia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestione tipografia</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_impiegato_connesso">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>90</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Impiegato connesso: &lt;nome_cognome_impiegato&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_coda_stampa">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualizza coda di stampa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gestione_prodotti">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestione prodotti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_ordini">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualizza ordini in corso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gestione_utenti">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestione utenti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run_backup">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Esegui backup del database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
